pyproject.toml

```diff
@@ -1,3 +1,37 @@
+[build-system]
+requires = ["setuptools>=61.0", "wheel", "setuptools_scm[toml]>=6.2", "libsass"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "nav"
+description = "Network Administration Visualized - A comprehensive, free Network Management System"
+authors = [{name="Uninett AS", email="nav-support@uninett.no"}]
+maintainers = [{name="Uninett Opensource", email="opensource@uninett.no"}]
```

Comment:
I just checked (since I don't trust my faulty memory), and nav-support@uninett.no is already redirecting straight into the main queue of the SSC.  Don't know about the other address, though.

Reply:
https://packaging.python.org/en/latest/specifications/core-metadata/#core-metadata-maintainer suggests that the maintainer attribute is superfluous if the maintainer and the author are basically the same.  I therefore suggest removing it entirely, and replacing this section like this:

```suggestion
authors = [{name="Sikt - Norwegian agency for shared services in education and research", email="nav-support@uninett.no"}]
```